df.history.xml

```diff
@@ -793,6 +793,34 @@
         <int32_t init-value='-1'/>
     </struct-type>
 
+    <enum-type type-name='identity_type' base-type='int32_t'>
+         <enum-attr name='tag' comment='for identity.id'/>
+         
+         <enum-item name='HidingCurse' comment="Inferred from Units.cpp after examining code using 'unk_4c'">
+            <item-attr name='tag' value='history'/>
```

Comment:
For future reference, the value should match the name of the union field that's valid for this enum item (i.e. histfig_id or nemesis_id). Apparently tagged union support (mainly in check-structures-sanity) doesn't handle unnamed unions yet, though, so it's not breaking anything.

We seem to have around 9 other unnamed unions that could be addressed, so no need to worry about it in this PR. [Example](https://github.com/DFHack/df-structures/blob/353caa3413f728938821c8aaccebb389a322bfef/df.ui.xml#L33-L36)

Reply:
I totally agree it should match. The mismatch is the result of failing to change these values when moving to the unnamed union (an explanation, not an excuse).